psutil/_pslinux.py

```diff
@@ -883,6 +884,12 @@ def cmdline(self):
         return [x for x in data.split('\x00')]
 
     @wrap_exceptions
+    def environ(self):
+        with open_text("/proc/%s/environ" % self.pid) as f:
```

Comment:
This has to be `with open_text("%s/%s/environ" % (get_profs_path(), self.pid)) as f:`